### PR TITLE
deleteAdmin Api fix issue #952

### PR DESCRIPTION
**Pull request by SwayamRana808**
## Issue that this pull request solves

 Closes: #952 

## Proposed changes
deleteFaq api expecting adminId fix
### Brief description of what is fixed or changed
deleteFaq api expecting adminId fix
## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/pages/Admin/Components/Faq/AddFaq/AddFaq.jsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ export class AddFaq extends React.Component {
     e.preventDefault();
     if (this.handleValidation()) {
       let tags = this.state.tags;
-      return fetch(`${END_POINT}/faq`, {
+      return fetch(`${END_POINT}/faq/faq`, {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Faq/AddFaq/AddFaq.jsx:99]: [question] shouldn't it be `getFaq`?

**Review by Kajol-Kumari — COMMENTED** [review]
can you please add screenshot/video testing the functionality?

File: frontend/src/pages/Admin/Components/Faq/AddFaq/AddFaq.jsx (hunk 1 above)
Comment (SwayamRana808) [reply frontend/src/pages/Admin/Components/Faq/AddFaq/AddFaq.jsx:99]: `/faq`  route  for post method to add faq and i just added base` /faq` route  whereas` /getFaq` is for get Method/
```
const faq = require('./post');
const getfaq = require('./getFaq');
router.post('/faq', validation(FAQValidationSchema), faq);
router.get('/getFaq', getfaq);

```

Comment (SwayamRana808) [reply frontend/src/pages/Admin/Components/Faq/AddFaq/AddFaq.jsx:99]: base route 
```
const faq = require('./faq');
`router.use('/faq', faq);`
```

**Comment by SwayamRana808** [comment]
**Initially two faqs cards ---getFaq call**
Screenshot 2024-05-26 130015
**calling deleteFaq without auth--**
Screenshot 2024-05-26 130044
**calling deleteFaq with auth and superAdmin true**
Screenshot 2024-05-26 130112
**Calling again getFaq now there are 1 faq card--**
Screenshot 2024-05-26 130139

File: frontend/src/pages/Admin/Components/Faq/AddFaq/AddFaq.jsx (hunk 1 above)
Comment (Kajol-Kumari) [reply frontend/src/pages/Admin/Components/Faq/AddFaq/AddFaq.jsx:99]: can we please make it `/postFaq` instead as `faq/faq` looks confusing

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**